MapleServer2/Packets/ClubPacket.cs

```diff
@@ -50,11 +97,11 @@ public static Packet CreateClub(Party party, Club club)
             pWriter.WriteInt();
             pWriter.WriteInt();
             pWriter.WriteLong();
-            pWriter.WriteByte((byte)party.Members.Count);
+            pWriter.WriteByte((byte)party.Members.Count); //loop
 
             foreach (Player member in party.Members)
             {
-                pWriter.WriteByte((byte)party.Members.Count);
+                pWriter.WriteByte(0x2);
```

Comment:
So this is always 0x2 no matter what?

Reply:
Yeah, I was testing this and it appears that 0x2 is indicating the start of a loop, not the amount of times it has to loop/which loop it's at.